plugin/spotify.vim

```diff
@@ -305,4 +305,5 @@ call s:SpotifyClearCache()
 let s:SpotifyOpenTrackUri = function( s:GetOS() . "UriOpener")
 
 command! -nargs=* Spotify call SpotifyTrackSearch("<args>")
+command! -nargs=* S call SpotifyTrackSearch("<args>")
```

Comment:
If you're not sure about using S as the shorter command, please suggest what should be used instead and I'll push a commit changing the command.